local_setup.md

```diff
@@ -0,0 +1,193 @@
+
+1.  Click on "Show User Security Credentials" and you see your needed keys. **IMPORTANT: **Download and store them in a secure place.
+
+1.  Now click on your newly created IAM User and navigate to "Permissions". There you open the "Inline Policies" by clicking on the arrow beside it (if its not already opened)
+
+  ![Inline Policies](more-around-inline-policies.png)
+1.  Now click on the "click here" link in the "Inline Policies" section. 
+
+1.  Choose "Custom Policy" and click select.
+
+  ![Select Custom Policy](select-custom-policy.png)
+
+1.  Set the name of your Policy to something like: ```<yourname>-s3-policy```. 
+
+1.  Insert this text to "Policy Document:
+```json
+{
+  "Statement": [
+    {
+      "Action": [
+        "s3:ListAllMyBuckets"
+      ],
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::*"
+    },
+    {
+      "Action": "s3:*",
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::<your-s3-bucket-name>"
+    },
+    {
+      "Action": "s3:*",
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::<your-s3-bucket-name>/*"
+    }
+  ]
+}
+```
+It should look like this:
+
+  ![New Policy](insert-new-policy.png)
+
+1.  Click on "Apply Policy"
+
+### Configure renuo-upload-signing
+
+Now you have to use the newly created bucket in your renuo-upload-signing by setting following keys in the *.env*-File:
+
+-  **S3_BUCKET_NAME:** needs the name you gave to your bucket.
+-  **CDN_HOST** differs if you changed the bucket-location for example. The part after the ```/``` will always be your bucket-name though.
+-  **S3_PUBLIC_KEY** needs the "Access Key ID" which you downloaded before.
+-  **S3_SECRET_KEY** needs the "Secret Access Key" which you downloaded before.
+
+The *.env* file exists after running `bin/setup` as a copy of *example.env*. Adjust it:
+
+```rb
+S3_BUCKET_NAME: 'renuo-upload-<yourname>-development'
+S3_PUBLIC_KEY: 'your public key found in Amazon IAM'
+S3_SECRET_KEY: 'your secret key found in Amazon IAM'
+CDN_HOST: 's3.eu-central-1.amazonaws.com/renuo-upload-<yourname>-development' #without https://, just the domain
+```
+
+### Set Up renuo-cms-demo as Example Project
+
+1.  Clone it to your local machine:
+```
+git clone git@github.com:renuo/renuo-cms-demo.git
+```
+1.  Configure it in renuo-upload-signing by adding this line to the file ```config/.env```:
+```rb
+API_KEYS: {"key":"h8934hghd389g89fh98h","app_name":"renuo-cms-demo","env": "development"}
+```
+
+1.  Configure it in renuo-cms-api. Open the console with ```rails c``` and write:
+```rb
+CredentialPair.create(
+ private_api_key: "47DTrw46jNDtt53g56Hg5MMt5",
+ api_key: "T3i1A247Rd1",
+ project_name: "renuo-cms-demo",
+ renuo_upload_api_key: "h8934hghd389g89fh98h",
+ renuo_upload_signing_url: "http://renuo-upload-signing.dev:3003/generate_policy"
+)
+```
+
+1.  Navigate to the ```index.html``` of renuo-cms-demo and make a search/replace with:
+
+  -  search for ```https://renuo-cms-api-demo.herokuapp.com/``` 
```

Comment:
this step is a bit cumbersome. can we change that? e.g. create a new file which is used only here?